assets/sass/components/_posts.scss

```diff
@@ -13,14 +13,32 @@
 		@include vendor('flex-wrap', 'wrap');
 		margin: 0 0 _size(element-margin) ($gutter * -1);
 		width: calc(100% + #{$gutter});
-
+		color: _palette(fg-bold);
+		
+		h3 {
+			margin-top: 1em;
+		}
```

Comment:
what is this for?

Reply:
this is for the cards on the homepage to add space between the image and h3 text.
![Screenshot 2023-12-13 215306](https://github.com/Yu-HuanWu/mandybbakes/assets/41420965/59bcf822-a751-408a-b142-8a3d68d35b36)